### ISSUE TITLE
Please explain plugin usefulness

### ISSUE THREAD

**Issue by pcav** [jagodki/Batch-Interpolation#1]
The functionality described in the About seems already present in core QGIS.
Please clarify what the plugin is actually adding.
Sorry for bothering, but we are trying hard not to duplicate functionality, so to reduce confusion for the users and administrative burden.
Thanks.

**Comment by jagodki** [comment]
Dear Paolo,

your right that I used functionalities already present in QGIS (v2). The advantage of this Plugin is the following:
I am starting a project where interpolation of point data and calculation of contour lines has to be happen for multiple times. I tried to create a batch process of it and a process with the graphic modeler. But the problem is, that the QGIS-interpolation (located in the Raster-menu) is not available in the processing toolbox. I could use interpolation methods like IDW, Kriging, Spline etc. using SAGA, GRASS or GDAL, but the usage of a TIN/Delaunay Triangulation to interpolate a DEM is mandatory. I.e. it is not known to me, that I could start a batch process of TIN interpolations in QGIS 2 without programming in python. My plugin has the advantage, that the user just need to choose a point layer and select all the attributes, which should be used for an own interpolation. To start the interpolation tool manually for 10 or 100 attributes is not very handy. With my plugin, it is just a little initialisation needed and after this the calculations of all attributes runs automatically. So I think, my plugin provides a functionality which is missing in QGIS 2.

Sincerely,
Christoph
